tools/c7n_mailer/c7n_mailer/handle.py

```diff
@@ -40,11 +41,13 @@ def config_setup(session):
     return config
 
 
-def start_c7n_mailer(event, context, logger):
+def start_c7n_mailer(event, context, logger, config=None, parallel=False):
```

Comment:
allow someone who is calling the mailer directly as a function (instead of lambda) to decide to use parallelization, parallel is False by default, so it will run in the normal threaded mode unless you decide to do multiprocessing explicitly.